utils/cliutils/utils_test.go

```diff
@@ -379,3 +379,60 @@ func (t *redirectingTransport) RoundTrip(req *http.Request) (*http.Response, err
 	// For all other requests, use the base transport
 	return t.baseTransport.RoundTrip(req)
 }
+
+// TestGetHasDisplayedSurveyLink tests the survey link environment variable check with parametrized test cases
+func TestGetHasDisplayedSurveyLink(t *testing.T) {
+	// Save original environment variable value
+	originalValue := os.Getenv(JfrogCliHideSurvey)
+	defer func() {
+		// Restore original value
+		if originalValue == "" {
+			os.Unsetenv(JfrogCliHideSurvey)
+		} else {
+			os.Setenv(JfrogCliHideSurvey, originalValue)
+		}
+	}()
+
+	testCases := []struct {
+		name       string
+		envValue   string
+		shouldHide bool
+	}{
+		{
+			name:       "env_var_not_set",
+			envValue:   "", // This will be handled by unsetting the env var
+			shouldHide: false,
+		},
+		{
+			name:       "env_var_true",
+			envValue:   "true",
+			shouldHide: true,
+		},
+		{
+			name:       "env_var_bad_input",
+			envValue:   "garbage",
+			shouldHide: false,
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			// Set up environment variable
+			if tc.envValue == "" {
+				os.Unsetenv(JfrogCliHideSurvey)
+			} else {
+				os.Setenv(JfrogCliHideSurvey, tc.envValue)
```

Comment:
Golang now has a test function for this: https://pkg.go.dev/testing#T.Setenv
```suggestion
				t.Setenv(JfrogCliHideSurvey, tc.envValue)
```